SECURITY.md

```diff
@@ -0,0 +1,41 @@
+# Security Policy
+
+Full details of the Automattic Security Policy can be found on [automattic.com](https://automattic.com/security/).
+
+## Supported Versions
+
+### Web
+
+Only the latest version of Calypso is supported.
```

Comment:
I'm not completely sure what this means. Calypso doesn't really have versions, so can we make this more accurate somehow?

Reply:
I wasn't sure about this either... What's the best way to say we only support the currently live version of Calypso?

Reply:
Maybe we could reference the `latest version in trunk` here? That would make sense. And we could further clarify it with something like "Calypso is continuously deployed shortly after each merge, multiple times per day."

Reply:
Done.